Odata-docs/client/v6/deal-with-server-driven-paging.md

```diff
@@ -19,15 +19,127 @@ Example:
 var context = new DefaultContainer(new Uri("https://services.odata.org/v4/TripPinServiceRW/"));
 
 // DataServiceQueryContinuation<T> contains the next link
-DataServiceQueryContinuation<Person> token = null;
+DataServiceQueryContinuation<Person> nextLink = null;
 
 // Get the first page
 var response = await context.People.ExecuteAsync() as QueryOperationResponse<Person>;
 
 // Loop if there is a next link
-while ((token = response.GetContinuation()) != null)
+while ((nextLink = response.GetContinuation()) != null)
 {
     // Get the next page
-    response = await context.ExecuteAsync<Person>(token) as QueryOperationResponse<Person>;
+    response = context.Execute<Person>(nextLink);
```

Comment:
Why the change from `ExecuteAsync()` to `Execute()`?

Reply:
#Resolved